frontend/test/e2e/ContractExampleWizardOfCoin.spec.ts

```diff
@@ -1,26 +1,47 @@
 import { WebDriver, By, until } from 'selenium-webdriver'
 import { ContractsPage } from '../pages/ContractsPage.js'
 import { RunDebugPage } from '../pages/RunDebugPage.js'
+import { SettingsPage } from '../pages/SettingsPage.js'
 import { before, describe, after, it } from 'node:test'
 import { expect } from 'chai'
 import { getDriver } from '../utils/driver.js'
 
 let driver: WebDriver
 let contractsPage: ContractsPage
 let runDebugPage: RunDebugPage
+let settingsPage: SettingsPage
 
 describe('Contract Example WizardOfCoin', () => {
   before(async () => {
     driver = await getDriver()
     await driver.manage().setTimeouts({ implicit: 10000 })
     contractsPage = new ContractsPage(driver)
     runDebugPage = new RunDebugPage(driver)
+    settingsPage = new SettingsPage(driver)
```

Comment:
@bradleySuira please make this piece of code reusable in some /common folder

Reply:
done @cristiam86 added a page factory and the driver util has the common settings but we should create an instance per test